locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -284,7 +292,7 @@ msgstr "Português Europeu (Portugiesisch)"
 
 #: bookwyrm/settings.py:258
 msgid "Swedish (Svenska)"
-msgstr ""
+msgstr "Swedish (Schwedisch)"
```

Comment:
Not related to the main part of the PR, but this called it to my attention: shouldn't both of these be the other way around? All the other languages are "Native Spelling (Translated Spelling)", so I think these should be "Svenska (Swedish)" and "Svenska (Schwedish)"

Reply:
Good catch -- https://github.com/bookwyrm-social/bookwyrm/pull/1901